lms/djangoapps/teams/api.py

```diff
@@ -278,3 +278,28 @@ def get_team_for_user_course_topic(user, course_id, topic_id):
             membership__user__username=user.username,
             topic_id=topic_id,
         ).first()
+
+
+def get_team_anonymous_user_ids(user, team):
+    """ Get the anonymous user IDs for members of a team, used in team submissions
+        Requesting user must be a member of the team or course staff
+
+        Returns:
+            (Array) User IDs, sorted to remove any correlation to usernames
+    """
+    if not user or not team:
+        raise Exception("User and team must be provided for ID lookup")
+
+    if not has_course_staff_privileges(user, team.course_id) and not user_is_a_team_member(user, team):
+        raise Exception("User {user} is not permitted to access team info for {team}".format(
+            user=user.username,
+            team=team.team_id
+        ))
+
+    ids = []
+    for team_member in team.users.all():
+        ids.append(anonymous_id_for_user(user=team_member, course_id=team.course_id, save=False))
+
+    ids.sort()
+
+    return ids
```

Comment:
It might be more helpful to return a dictionary keyed by user id and valued by that user's anonymous id.

Reply:
One discussion we had was around intentionally sorting/randomizing user IDs so an end-user can't establish connection between anonymous ID and real user ID.

I don't know if providing that extra correlation could be abused in any way... if not, I'm fine to switch to a dict.

@jansenk and @mattcarter, thoughts?

Reply:
Ah, you even mentioned that in the docstring.  In that case, that's fine.  Small nit: this could be compacted into
```python
return sorted([
    anonymous_id_for_user(user=team_member, course_id=team.course_id, save=False)
    for team_member in team.users.all()
])
```